ProjectLighthouse.Servers.Website/Controllers/Moderator/ModerationSlotController.cs

```diff
@@ -68,4 +68,23 @@ public async Task<IActionResult> DeleteLevel([FromRoute] int id)
 
         return this.Redirect("~/slots/0");
     }
+
+    [HttpGet("flag")]
+    public async Task<IActionResult> FlagLevel([FromRoute] int id)
+    {
+        UserEntity? user = this.database.UserFromWebRequest(this.Request);
+        if (user == null) return this.StatusCode(403);
```

Comment:
Probably better to just redirect back to the home page or the slot page since this isn't used as an xhr request